.github/workflows/cicd.yml

```
@@ -261,6 +261,24 @@ jobs:
       code_coverage: false
       configurations: '["NativeOnlyDebug", "NativeOnlyRelease"]'
 
+  driver_native_only_arm64_tests:
+    needs: regular_native-only
+    if: github.repository == 'microsoft/ebpf-for-windows' && (github.event_name == 'schedule' || github.event_name == 'pull_request' || github.event_name == 'push' || github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch')
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      build_platform: 'ARM64'
+      pre_test: .\setup_ebpf_cicd_tests.ps1 -KmTracing $true -KmTraceType "file" -ExecuteOnHost -Architecture arm64
+      test_command: .\execute_ebpf_cicd_tests.ps1 -TestMode "CI/CD" -ExecuteOnHost
+      post_test: .\cleanup_ebpf_cicd_tests.ps1 -KmTracing $true -ExecuteOnHost
+      name: driver_native_only_arm64
+      build_artifact: Build-ARM64-native-only
+      environment: '["windows-11-arm"]'
+      # driver test copies dumps to testlog folder.
+      gather_dumps: false
+      # driver tests manually gather code coverage
```

```suggestion
      # Driver test copies dumps to testlog folder.
      gather_dumps: false
      # Driver tests manually gather code coverage.
```
nit: other comments start with a capital letter and end in a period, so better to use consistent style.